sources/packages/web/src/services/http/ApplicationExceptionApi.ts

```diff
@@ -38,4 +42,37 @@ export class ApplicationExceptionApi extends HttpBaseClient {
       payload,
     );
   }
+
+  /**
+   * Gets all pending student application exceptions.
+   * @param paginationOptions options to execute the pagination.
+   * @returns list of student application exceptions.
+   */
+  async getPendingExceptions(
+    paginationOptions: PaginationOptions,
+  ): Promise<PaginatedResultsAPIOutDTO<ApplicationExceptionSummaryAPIOutDTO>> {
+    let url = `application-exception`;
+    // Adding pagination params. There is always a default page and pageLimit for paginated APIs.
+    url = addPaginationOptions(
```

Comment:
**Not part of this PR** but the code from line 56 till 73 should be moved to a helper near `addPaginationOptions` and `addSortOptions` and just receive the URL + paginationOptions.